deepchem/models/keras_model.py

```diff
@@ -181,14 +186,41 @@ def __init__(self,
       self.optimizer = optimizer
     self.tensorboard = tensorboard
 
-    # W&B logging
+    # W&B flag support (DEPRECATED)
+    if wandb:
+      logger.warning(
+          "`wandb` argument is deprecated. Please use `wandb_logger` instead. "
+          "This argument will be removed in a future release of DeepChem.")
```

Comment:
TorchModel should also display this warning.

Reply:
Added! Must've missed it earlier